src/lib/util/Cache/UserTags.ts

```diff
@@ -56,17 +61,22 @@ export class UserTags extends Collection<string, UserTag> {
 	}
 
 	public async fetchEntry(id: string) {
-		const existing = super.get(id);
-		if (typeof existing !== 'undefined') return [id, existing] as const;
-
-		const user = await this.client.users.fetch(id);
-		return [id, this.create(user)] as const;
+		const userTag = await this.fetch(id);
+		return [id, userTag] as const;
 	}
 
 	public static get [Symbol.species](): CollectionConstructor {
 		return Collection as unknown as CollectionConstructor;
 	}
 
+	private requestHandlerGet(id: string) {
+		return this.client.users.fetch(id);
```

Comment:
Same as above

Reply:
Uh... the original code doesn't catch any errors at all, and that goes out of this PR's scope.

Reply:
Also, RequestHandler's functions require objects with a property of `id`, when an error matches any of the codes given to `resolveOnErrorCodes`, it returns `null`, which doesn't match the trait `IdKeyed` from `@klasa/request-handler`.